serving-core/src/main/scala/com/stratio/sparkta/serving/core/constants/AppConstant.scala

```diff
@@ -27,25 +27,28 @@ object AppConstant {
   final val ClasspathJarFolder = "repo"
   final val ClusterExecutionJarFolder = "driver"
   final val ExecutionMode = "executionMode"
-  final val DefaultExecutionMode = "local"
   final val ConfigAppName = "sparkta"
   final val ConfigApi = "api"
   final val ConfigHdfs = "hdfs"
   final val ConfigDetail = "config"
   final val ConfigLocal = "local"
-  final val ConfigStandAlone = "standAlone"
+  final val ConfigStandAlone = "standalone"
```

Comment:
Note that changing this value also implies changes in conf files and documentation